packages/patternfly-4/react-core/src/demos/BulkSelectTable.md

```diff
@@ -0,0 +1,225 @@
+---
+title: 'Bulk select table'
+section: 'demos'
+---
+
+## Bulk select table demo
+
+import { Pagination, PaginationVariant, Title, Checkbox } from '@patternfly/react-core';
+import { Table, TableHeader, TableBody} from '@patternfly/react-table';
+
+```js
+import React from 'react';
+import { Pagination, PaginationVariant, Title, Checkbox } from '@patternfly/react-core';
+import { Table, TableHeader, TableBody } from '@patternfly/react-table';
+
+class BulkSelectTableDemo extends React.Component {
+  constructor(props) {
+    this.state = {
+      res: [],
+      perPage: 20,
+      total: 100,
```

Comment:
Does `total` need to be a state property? Might be good to only capture properties that need to change in the component state.